documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,150 @@
+         2. The two competitions take place in different countries.
+   4. 3x3x3 Fewest Moves simultaneous competitions
+      1. 3x3x3 Fewest Moves simultaneous competitions are WCA competitions having only the 3x3x3 Fewest Moves event and taking place simultaneously in different locations.
+         1. Any location must be announced at least 28 days (four weeks) before 00:00 UTC of the start date of the competition on the WCA website.
+         2. If any location of such competitions is closer than 100 km to any other competition, then
+            1. both competitions should be at least 5 days apart, or
+            2. the other competition must not have the 3x3x3 Fewest Moves event.
+      2. A 3x3x3 Fewest Moves simultaneous competition cannot be part of any other regular competition.
+         1. Time frames when both are held cannot overlap.
+   5. Twin competitions
+      1. Twin competitions are WCA Competitions that are held less than 5 days apart for which the distance between them is less than 100 km, with the intention to accommodate more competitors to compete in one area.
+         1. Twin competitions may be accepted if the need for competitions in an area is so great that the community will benefit from having multiple competitions held in a short period of time.
+      2. Twin competitions are obliged to clearly and publicly state their condition of twin competitions and their special participation terms.
+         1. A competitor is only permitted to participate in one of the twin competitions.
+            1. If a competitor competes at more than one of the involved twin competitions, only the results from the competition that they compete in first will be kept.
+3. WCA Delegate(s)
+4. Organizer(s)
+   1. Organizers can be specific persons and/or organizations.
+5. Competitor limit
+   1. The competitor limit can be on either a "first come first serve" basis or based upon qualification results or rankings in the WCA world rankings of a previously announced calendar date.
+      1. Qualification for an event must be based only on results from the same event.
+      2. Qualification for an event must be based on one type of result only (i.e. average/mean or single)
+         1. Exception: in case of using average/mean results, single results must be used to break ties.
+   2. If the competitor limit for a competition or an event is reached before the competition, the organization team must not approve any additional registrations for the competition or the event.
+   3. The competitor limit is per competition. Per day competitor limits are not allowed.
+   4. The WCA Competition Announcement Team will review the competitor limit and/or qualification results or rankings to ensure that they are utilized properly.
+6. Registration requirements
+   1. The registration requirements must be clear so that any competitor from anywhere in the world knows how to and has the ability to register.
```

Comment:
Ouch, this sentence definitely has to change, as it is literally impossible to fulfill.